receiver/opentelemetryreceiver/opentelemetry.go

```diff
@@ -0,0 +1,270 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opentelemetryreceiver
```

Comment:
I think `otlpreceiver` as package/directory name is shorter and no less descriptive.

Reply:
On the Collector SIG call we had discussed that this receiver would be a temporary one. Is this  (otlpreceiver) going to be temporary? OR is it just the part (ot_to_oc conversion) of the receiver that will be temporary? If it is former then otlpreceiver name should be reserved for the permanent receiver and maybe name this one to something else.

Reply:
The implementation will be temporary. It will be rewritten as soon as we have new internal data structures. However the receiver itself is not temporary, it will be a working receiver implement OTLP. It will be refactored, we will not create a new one.

Reply:
My interpretation is that only the "translator" part is temporary, since OpenCensus data types used internally will be replaced soon.

Reply:
Correct. There may be minor changes outside the translator but primary changes will be in translator.